sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -500,6 +500,8 @@ def wrap(func):
 
     removed_arg_names = removed_args if removed_args is not None else []
 
+    # TODO: Better handle position only arguments if they are ever a thing
```

Comment:
Sounds like we could remove this TODO. If you think we should keep it, would you mind expanding on what 'better' would mean? Thanks!

Reply:
Removed the TODO